classes/shortcodes.php

```diff
@@ -68,10 +68,11 @@ function get_shortcode_tags() {
 				'phone',
 				'fax',
 				'email',
-				'url'
+				'url',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->